[Docu-3170] Support for scoping plugins by consumer group. (#5874)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/consumer-groups/index.md

---
title: Consumer Groups
badge: enterprise
---

> Consumer Groups is a powerful feature available in {{site.base_gateway}} that enables the organization and categorization of consumers (users or applications) within an API ecosystem. By grouping consumers together, you can manage in an aggregated and more efficient manner. Consumer groups functionality has the capability to scope plugins specifically to defined groups, making configurations and customizations more flexible and convenient. Consumer groups eliminate the need to manage consumers individually, providing a scalable approach to managing configurations.

Reviewer: ```suggestion
Consumer groups enable the organization and categorization of consumers (users or applications) within an API ecosystem. By grouping consumers together, you eliminate the need to manage them individually, providing a scalable, efficient approach to managing configurations. 

With consumer groups, you can scope plugins specifically to defined groups, making configurations and customizations more flexible and convenient. 
```
It got a bit repetitive in the middle. Also combined a couple of similar sentences.

{% if_version gte:3.4.x %}

> ### Use cases

Reviewer: ```suggestion
## Use cases
```

> * Managing permissions:   Consumer Groups can be used to define different sets of users with varying levels of permissions. For example, you can create distinct consumer groups for regular users, premium users, and administrators.

Reviewer 2: ```suggestion
* Managing permissions: Consumer groups can be used to define different sets of users with varying levels of permissions. For example, you can create distinct consumer groups for regular users, premium users, and administrators.
```

* Managing roles: Within an organization, there may be various departments or teams that interact with APIs differently. By creating consumer groups for these different roles, you can customize the API usage experience. For instance, an organization could have separate consumer groups for the marketing team, development team, and support team.

* Resource quotas and rate limiting:  Consumer groups can be used to enforce resource quotas and rate limiting on different sets of consumers. For instance, you can apply different rate limits to different consumer groups based on their subscription plans. 

> * Customizing plugin configurations: With the ability to scope plugins specifically to defined groups, different Consumer Groups can have distinct plugin configurations based on their requirements. For example, one group may require additional request transformations while another may not need them at all.

Reviewer 2: ```suggestion
* Customizing plugin configurations: With the ability to scope plugins specifically to defined groups, different consumer groups can have distinct plugin configurations based on their requirements. For example, one group may require additional request transformations while another may not need them at all.
```

    HTTP/1.1 204 No Content
    ```

{% endif_version %}

> ## More Information

Reviewer: ```suggestion
## More information
```
